### DISCUSSION TITLE
Typed Modules in Monorepo

### DISCUSSION THREAD

**Discussion by guilhermedelyra** [pdm-project/pdm#2420] [Q&A]
Hi all!

I've created a monorepo and don't know how to indicate to mypy that the packages I'm importing does exists (and are typed)...

when i run mypy from the root module, i get

"module is installed, but missing library stubs or py.typed marker [import-untyped]"

**Comment by sanmai-NL** [comment]
@frostming Do you have a solution/work-around for this?
